plugin.py

```diff
@@ -101,3 +108,42 @@ def get_default_attr_name_case(configuration: ClientConfig) -> str:
     if preferred_attr_name_case == 'camel':
         return 'camelCase'
     return 'kebabCase'
+
+
+def get_language_features(configuration: ClientConfig, is_main_server: bool) -> dict:
+    use_second_server = configuration.settings.get('volar.vueserver.useSecondServer')
+    main_language_features = {
+        "references": True,
+        "implementation": True,
+        "definition": True,
+        "typeDefinition": True,
+        "callHierarchy": False,
+        "hover": True,
+        "rename": True,
+        "renameFileRefactoring": False,
+        "signatureHelp": True,
+        "codeAction": True,
+        "workspaceSymbol": True,
+        "completion": {
+            "defaultTagNameCase": get_default_tag_name_case(configuration),
+            "defaultAttrNameCase": get_default_attr_name_case(configuration),
+            "getDocumentNameCasesRequest": False,
+            "getDocumentSelectionRequest": False,
+        },
+        "schemaRequestService": False
+    }
+    second_language_features = {
+        "documentHighlight": True,
+        "documentLink": True,
+        "codeLens": {"showReferencesNotification": True},
+        "semanticTokens": True,
+        "inlayHints": False,
+        "diagnostics": True,
+        "schemaRequestService": False
+    }
+    if is_main_server:
+        all_language_features = {}
+        all_language_features.update(main_language_features)
+        all_language_features.update(second_language_features)
```

Comment:
I tried to use spread (`c = {**a, **b}`) instead of `.update`:

```
all_language_features = {**main_language_features, **second_language_features}
```
but I didn't know that Python3.3 does not support that, and it throws a syntax error.

I saw in the LSP repo this code and it doesn't throw:
```
    return {**jsonrpc(), "id": request_id, "error": err.to_lsp()}
```